app/admin/carts.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register Cart do
+  permit_params :status
```

Comment:
since we permit only one parameter, we should define form for this model, which contains only `status` field.

Reply:
Allowed all fields to be changed by Admin